app/controllers/ships_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ShipsController < ApplicationController
+
+  def index
+  end
+
+  def create
+    @ship = Ship.create(ship_params)
+    render action: "index"
```

Comment:
Why don't we redirect here instead of rendering the page (this makes it work better if someone refreshes the page)